physics/ephemeris_test.cpp

```diff
@@ -673,95 +673,117 @@ TEST_F(EphemerisTest, Спутник1ToСпутник2) {
+      {SolarSystemFactory::Miranda, 1e-8},
+      {SolarSystemFactory::Moon, 1e-8},
+      {SolarSystemFactory::Rhea, 1e-8},
       {SolarSystemFactory::Triton, 1e-8},
-      {SolarSystemFactory::Uranus, 1e-8},
+      {SolarSystemFactory::Ariel, 1e-9},
       {SolarSystemFactory::Callisto, 1e-9},
       {SolarSystemFactory::Charon, 1e-9},
+      {SolarSystemFactory::Iapetus, 1e-9},
       {SolarSystemFactory::Jupiter, 1e-9},
       {SolarSystemFactory::Mars, 1e-9},
       {SolarSystemFactory::Pluto, 1e-9},
+      {SolarSystemFactory::Sun, 1e-9},
+      {SolarSystemFactory::Titan, 1e-9},
+      {SolarSystemFactory::Vesta, 1e-9},
+      {SolarSystemFactory::Ceres, 1e-10},
+      {SolarSystemFactory::Oberon, 1e-10},
+      {SolarSystemFactory::Umbriel, 1e-10},
+      {SolarSystemFactory::Saturn, 1e-10},
+      {SolarSystemFactory::Titania, 1e-11},
+      {SolarSystemFactory::Uranus, 1e-11},
       {SolarSystemFactory::Neptune, 1e-12},
       {SolarSystemFactory::Eris, 1e-13}};
   static std::map<SolarSystemFactory::Index,
                   double> const expected_velocity_error = {
+      {SolarSystemFactory::Phobos, 1e-1},
+      {SolarSystemFactory::Enceladus, 1e-2},
+      {SolarSystemFactory::Mimas, 1e-2},
+      {SolarSystemFactory::Deimos, 1e-3},
       {SolarSystemFactory::Dione, 1e-3},
       {SolarSystemFactory::Io, 1e-3},
       {SolarSystemFactory::Tethys, 1e-3},
-      {SolarSystemFactory::Ariel, 1e-4},
       {SolarSystemFactory::Europa, 1e-4},
-      {SolarSystemFactory::Oberon, 1e-4},
+      {SolarSystemFactory::Miranda, 1e-4},
       {SolarSystemFactory::Rhea, 1e-4},
-      {SolarSystemFactory::Titania, 1e-4},
       {SolarSystemFactory::Triton, 1e-4},
-      {SolarSystemFactory::Umbriel, 1e-4},
+      {SolarSystemFactory::Ariel, 1e-5},
       {SolarSystemFactory::Charon, 1e-5},
-      {SolarSystemFactory::Titan, 1e-5},
-      {SolarSystemFactory::Uranus, 1e-5},
       {SolarSystemFactory::Ganymede, 1e-6},
-      {SolarSystemFactory::Iapetus, 1e-6},
       {SolarSystemFactory::Mercury, 1e-6},  // NOTE(egg): General relativity.
-      {SolarSystemFactory::Moon, 1e-6},
       {SolarSystemFactory::Pluto, 1e-6},
-      {SolarSystemFactory::Saturn, 1e-6},
+      {SolarSystemFactory::Titan, 1e-6},
+      {SolarSystemFactory::Umbriel, 1e-6},
       {SolarSystemFactory::Callisto, 1e-7},
       {SolarSystemFactory::Earth, 1e-7},
+      {SolarSystemFactory::Iapetus, 1e-7},
       {SolarSystemFactory::Jupiter, 1e-7},
+      {SolarSystemFactory::Moon, 1e-7},
+      {SolarSystemFactory::Oberon, 1e-7},
+      {SolarSystemFactory::Saturn, 1e-7},
       {SolarSystemFactory::Sun, 1e-7},
       {SolarSystemFactory::Venus, 1e-7},
+      {SolarSystemFactory::Ceres, 1e-8},
       {SolarSystemFactory::Mars, 1e-8},
       {SolarSystemFactory::Neptune, 1e-8},
+      {SolarSystemFactory::Titania, 1e-8},
+      {SolarSystemFactory::Uranus, 1e-8},
+      {SolarSystemFactory::Vesta, 1e-8},
       {SolarSystemFactory::Eris, 1e-10}};
```

Comment:
Make an issue to rewrite this test using osculating elements (probably in the parent equator frame because of the properties of oblateness-induced precession), so we can figure out what's happening with Phobos (it's clearly some sort of angular error, but is it the argument or the longitude being wrong?), and so we don't have the utterly useless relative error in the solar-system-barycentric position.